### PR TITLE
Feature/chat title

### PR DESCRIPTION
**Pull request by sergiohs84**

### CODE REVIEW

**Comment by alxvasilev** [comment]
I confirm it works properly.

File: examples/megacli.cpp [hunk 1]
```diff
@@ -638,6 +678,13 @@ void DemoApp::printChatInformation(TextChat *chat)
     {
         cout << " no peers (only you as participant)" << endl;
     }
+    if (!chat->title.empty())
+    {
+        char tstr[chat->title.size() * 4 / 3 + 4];
```
Comment (javiserrano) [inline examples/megacli.cpp]: Local buffer of variable size. This probably won't compile on all platforms.

File: include/megaapi.h [hunk 2]
```diff
@@ -1397,6 +1395,13 @@ class MegaTextChat
      */
     virtual MegaHandle getOriginatingUser() const;
 
+    /**
+     * @brief getTitle Returns the title of the chat, if any.
+     *
```
Comment (javiserrano) [inline include/megaapi.h]: Comment if the returned pointer is owned by MegaTextChat or not.

**Comment by javiserrano** [comment]
All OK

**Pull request merged**